### ISSUE TITLE
/.kube/config: no such file or directory

### ISSUE THREAD

**Issue by nakfour** [opendatahub-io/opendatahub-operator#9]
The operator sometimes gives this error and does not install the components, it happens sometimes when installing multiple instances in multiple namespaces. It seems the issue is re-using the kustomize folder from the previous install, here is the log
```
msg="folder /tmp/kustomize exists, skip kustomize.Generate"
time="2020-04-27T17:20:19Z" level=info msg="/tmp/.cache/manifests exists; not resyncing "
```

There are no --force or --clean command to force kfctl to re-general the kustomize folder, read comments in this thread https://github.com/kubeflow/kubeflow/issues/4368

**Comment by vpavlin** [comment]
See https://github.com/kubeflow/kfctl/issues/242

**Comment by nakfour** [comment]
Temporary fix in PR #14
